src/devices/Nrf24l01/Nrf24l01.cs

```diff
@@ -0,0 +1,773 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Threading;
+
+namespace Iot.Device.Nrf24l01
+{
+    /// <summary>
+    /// Single chip 2.4 GHz Transceiver nRF24L01
+    /// </summary>
+    public class Nrf24l01 : IDisposable
+    {
+        private GpioController _gpio = null;
+        private SpiDevice _sensor = null;
+
+        private readonly int _ce;
+        private readonly int _irq;
+
+        private readonly byte[] _empty = new byte[0];
```

Comment:
should this be static

Reply:
also can use Array.Empty instead of new byte[0]

Reply:
In this case `Span<byte>.Empty` at the call site is probably the best choice. I believe this is only trying to pass an empty Span.

Reply:
To use Span<T>, you need to set it as a property. I will change it to Array.Empty